spaces/S000110/properties/P000022.md

```diff
@@ -3,9 +3,8 @@ space: S000110
 property: P000022
 value: true
 refs:
-- doi: 10.1007/978-1-4612-6290-9_6
-  name: Counterexamples in Topology
+-  doi: 10.1007/978-3-642-61701-0
+   name: General Topology (Bourbaki)
 ---
 
-Asserted in the General Reference Chart for space #113 in
-{{doi:10.1007/978-1-4612-6290-9_6}}.
+For any continuous $f: X \to \mathbb{R}$, the restriction $f\restriction_{\omega\cup \{F\}}$ is continuous for any $F\in M$. Since the topology on $\omega\cup \{F\}$ as a subspace of $X$ and of $\beta \omega$ coincides, and $\omega$ is dense in $\beta\omega$, it follows from theorem 1.8.5.1 of {{doi:10.1007/978-3-642-61701-0}} that $f$ is continuous as a function $f:\beta\omega\to \mathbb{R}$. Since {S108|P22}, $f$ is bounded.
```

Comment:
```suggestion
For any continuous $f: X \to \mathbb{R}$, the restriction $f\restriction_{\omega\cup \{F\}}$ is continuous for any $F\in M$. Since the topologies on $\omega\cup \{F\}$ as a subspace of $X$ and of $\beta \omega$ coincide, and $\omega$ is dense in $\beta\omega$, it follows from theorem I.8.5.1 of {{doi:10.1007/978-3-642-61701-0}} that $f$ is continuous as a function $f:\beta\omega\to \mathbb{R}$. Since {S108|P22}, $f$ is bounded.
```